ceph-dev-setup-new/config/definitions/ceph-dev-setup.yml

```diff
@@ -0,0 +1,59 @@
+- job:
```

Comment:
Would you change the filename here to ``ceph-dev-setup-new.yml``?  The config file for the ``ceph-dev-new`` job is also still named ``ceph-dev.yml``